src/lib/y2storage/disk.rb

```diff
@@ -68,6 +68,13 @@ def usb?
       transport.to_sym == :usb
     end
 
+    # Checks if it's in network
+    #
+    # @return [Boolean]
+    def network_trasporter?
+      transport.to_sym == :iscsi or transport.to_sym == :fcoe
```

Comment:
Please, use `||` instead of `or`. The second one if not an operator, is a control flow sentence.